src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { User } from '../../app.typedefs';
+
+interface Props {
+  user: User | null,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <>
+      <div data-cy="username">{`Name: ${user?.name}`}</div>
+      <div data-cy="email">{`Email: ${user?.email}`}</div>
```

Comment:
this is not the way to go. you cannot simply use ? everywhere
maybe it would be correct to not display the user at all if there is none